Cannot find module 'node:os'

<!--
👋 Hi, thanks for filing an issue on tslint-to-eslint-config! 💖
Please fill out all fields below to ensure your issue is addressed.

If your issue doesn't provide enough info to fully explain or reproduce your bug, it will be closed. 😦
-->

### 🐛 Bug Report

-   `tslint-to-eslint-config` version: 2.12.2
-   ESLint version: 8.10.0
-   Node version: 14.17.3

#### Actual Behavior

<!-- What is the behavior right now? -->

```
➤ npx tslint-to-eslint-config
npx: installed 64 in 3.749s
Cannot find module 'node:os'
Require stack:
- /Users/alukach/.npm/_npx/251/lib/node_modules/tslint-to-eslint-config/src/cli/runCli.js
- /Users/alukach/.npm/_npx/251/lib/node_modules/tslint-to-eslint-config/src/cli/main.js
- /Users/alukach/.npm/_npx/251/lib/node_modules/tslint-to-eslint-config/bin/tslint-to-eslint-config
```

#### Expected Behavior

<!-- What are the specific changes you'd like to see? -->

Conversion to work.

#### Reproduction
I'm assuming this is due to #1365

--- a/src/cli/runCli.test.ts
+++ b/src/cli/runCli.test.ts
@@ -1,6 +1,6 @@
 import { describe, expect, it, jest } from "@jest/globals";
-import path from "node:path";
 import { EOL } from "os";
+import path from "path";
 import { fileURLToPath } from "url";
 
 import { createStubLogger, expectEqualWrites } from "../adapters/logger.stubs";

--- a/src/cli/runCli.ts
+++ b/src/cli/runCli.ts
@@ -1,8 +1,8 @@
 import chalk from "chalk";
 import { Command } from "commander";
 import { promises as fs } from "fs";
-import { EOL } from "node:os";
-import path from "node:path";
+import { EOL } from "os";
+import path from "path";
 
 import { Logger } from "../adapters/logger";
 import { SansDependencies } from "../binding";

